packages/devtools/client/components/ServerRouteDetails.vue

```diff
@@ -133,7 +133,7 @@ const finalURL = computed(() => domain.value + finalPath.value)
 
 function parseInputs(inputs: any[]) {
   const formatted = Object.fromEntries(
-    inputs.filter(({ key, value }) => key !== undefined && value !== undefined).map(({ key, value }) => [key, value]),
```

Comment:
`key !== undefined` would let to:

![image](https://github.com/nuxt/devtools/assets/38922203/956348bd-c36e-4852-8373-944368891078)